plugin.xml

```diff
@@ -87,7 +87,7 @@ SOFTWARE.
               <source url="https://cdn.cocoapods.org/"/>
           </config>
           <pods>
-              <pod name="BranchSDK" spec="~> 2.2.0" />
+              <pod name="BranchSDK" spec="~> 3.0.0" />
```

Comment:
Could explicitly set it to 3.0.1, though since it's the first release on the 3.x series it won't make any difference.